geometry_msgs/Pose pose
geometry_msgs/Point[] corners_obb
float32 width
float32 height
float32 depth
bool is_gap
